blocks/core/ff_module/ff_module-profile-response-button/ff_module-profile-response-button.md

```diff
@@ -1,9 +1,32 @@
 <div data-ff_module-profile-response-button=""/>
 
-# Here be a profile-response button
+# Profile response button
 
 ## Markup 
 ```
 <div data-ff_module-profile-response-button=""/>
 ```
+### Props
 
+- `onSelect`: Function
+- `uiState`: String; css defined for `is-selected`, `is-updated` (default), `is-reviewed` in Melody
+- `guid`: String; unique id
+- `label`: String; Name of Recipient
+- `status`: String; Response Status
```

Comment:
No, the status is independent of the uiState. The status reflects the task status, the uiState only the state of the button (so its reviewed once the button has been clicked, updated if the status has changed but the button hasn't been clicked, selected is obvious, etc)